### PR TITLE
[Travis] fix webdriver

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes

Fix this PR https://github.com/aces/Loris/pull/6187

#### Testing instructions (if applicable)

Travis

#### Link(s) to related issue(s)
all PRs after https://github.com/aces/Loris/pull/6187

### CODE REVIEW

**Comment by lingma** [comment]
just say final try, then it will have to work

File: .phan/config.php [hunk 1]
```diff
@@ -34,10 +34,17 @@
     "unused_variable_detection" => false,
     "suppress_issue_types" => [
         "PhanTypePossiblyInvalidDimOffset",
-        "PhanUndeclaredMethod",
         "PhanTypeMismatchArgument",
         "PhanTypeMismatchProperty",
         "PhanTypeArraySuspiciousNullable",
+        "PhanTypeMismatchReturn",
```
Comment (driusan) [inline .phan/config.php:40]: We shouldn't be adding to the suppressed types in phan. Every one of these is a bug that's being caught by static analysis. I sent #6404 to revert the change to webdriver. I think after I merge #6404 this PR should be updated to revert that change and then fix this without regressions to our phan rules.

**Comment by kongtiaowang** [comment]
wrong ropo
